src/dsp/Shared/EQHelpers.cpp

```diff
@@ -0,0 +1,81 @@
+#include "EQHelpers.h"
+
+EQHelpers::EQHelpers()
+{
+}
+
+void EQHelpers::prepareToPlay (double sampleRate, int samplesPerBlock, int numChannels)
+{
+    SpectrumAnalyserTask.prepare (sampleRate, samplesPerBlock, numChannels);
+}
+
+void EQHelpers::processBlockInput (juce::AudioBuffer<float>& buffer)
```

Comment:
The original idea behind the `processBlockInput()` is that we would show different spectra for the input and output signals... not sure if we'd want to do that here?